app/models/user.rb

```diff
@@ -41,8 +41,12 @@ class User < ActiveRecord::Base
   has_many :mail_notifications, :through => :user_mail_notifications
 
   accepts_nested_attributes_for :user_mail_notifications, :allow_destroy => true, :reject_if => :reject_empty_intervals
-  attr_accessible :password, :password_confirmation
-  include AccessibleAttributes
+  attr_accessible :password, :password_confirmation, :login, :firstname,
```

Comment:
I think you're confusing terms - accessible only means that this field can be edited directly using update_attributes, etc. if the users#edit/new forms have those fields, they need to be accessible. if you wish to protect them from mass asignment, you need to write a specific form which does that, and uses the password= (etc) methods, instead of mass assigning those values.

those values "feel" like they need to be protected, but as long as we rely on mass assignment, we should use them.